proposals/policies/chair-decision-process.md

```diff
@@ -0,0 +1,9 @@
+# Web of Things (WoT) Policy
+## Chairs' Decision Process and Quorum
+- The following relates only to decisions that the WG agrees can be delegated to chairs, which includes the following:
+    - TBD, ex. rejecting unsolicited IE applications
+- Chairs' decisions will generally by consensus of all three chairs
+- Weekly meetings will be held just prior to main WoT working group call
+- If the chairs cannot meet consensus on a decision, and a decision needs to be made immediately, a majority vote (2/3) process will be used.
+- Consistent with the above, quorum is 2 out of 3 chairs.  If only one chair can make the chairs' meeting no decisions can be made, although some administrative work may take place (preparing agendas for the main call, etc).
+- A chair not attending a meeting will be considered to be abstaining from voting.  Consistent with the above, if only two chairs are present in a meeting and cannot agree on a decision, it will have to be deferred until the third chair can join.
```

Comment:
* Made some updates to allow a chair to email their decision.  We should discuss if this can include "delegating" their decision power to one of the other chairs.  I would rather not do this, however.
* Also made a change to consider IRC-only "attendance" as being permissible (as long as we can check that a chair agrees with a decision, basically).
* Note that there is still a "TBD" for the concrete list of decisions for which this process is applicable.  We need to go through the "roles" document and extract things that do NOT need membership review (or that technically only chairs can decide).